pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "freetype-py",
     "uharfbuzz",
     "jinja2",
+    "hsluv==5.0.0",
```

Comment:
We're rather careful about adding dependencies. I Checked this dependency: https://github.com/hsluv/hsluv-python: It's a single-module pure Python lib without further dependencies. IMO fine to add this for this purpose.

Reply:
We can make it optional? Don't need it unless you use all these color functions.

Reply:
can make it optional and raise an error in case it's being used

Reply:
We can also vendor the 300-line module?

Reply:
made it optional

Reply:
Agreed. We can eventually come back on this decision anyway. Let's add it as a dep (and not vendor)

Reply:
updated. made hsluv default for interpolation and ;ighter/darker

Reply:
updated. now it's a non-optional dependency